pkg/public/util.go

```diff
@@ -127,3 +129,14 @@ func ReconcileRedis(ctx context.Context, client client.Client, productName, depl
 
 	return r, nil
 }
+
+// ReconcileStrategyMaps to be used to reconcile strategy maps expected in RHMI installs
+// A single function which can check the infrastructure and provision the correct strategy config map
+func ReconcileStrategyMaps(ctx context.Context, client client.Client, tier, backupWindow, maintenanceWindow, namespace string) error {
```

Comment:
Is there any reason this public function isn't in `strategies.go`? They're both in the same package and it's probably a better home for it